app/src/main/java/co/jp/duonghb/MainActivity.kt

```diff
@@ -19,10 +19,10 @@ class MainActivity : ComponentActivity() {
             BitriseCIDemoTheme {
                 // A surface container using the 'background' color from the theme
                 Surface(
-                    modifier = Modifier.fillMaxSize(),
+                     modifier = Modifier.fillMaxSize(),
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nokaduonghb/BitriseCIDemo/blob/1f98f9c028efc46eb1b708d7f63473c12c483eba/app/src/main/java/co/jp/duonghb/MainActivity.kt#L22"></span>Unexpected indentation (21) (should be 20)</td>
    </tr>
  </tbody>
</table>